### PR TITLE
Homework2: Ivan Veremeichik

### PR DESCRIPTION
**Pull request by NorgeMan**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework2/task8.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+# HHeelllloo  WWoorrlldd
+# Input:
+# 1234!_
+# Output:
+# 11223344!!__
+# double_char("String") ==> "SSttrriinngg"
+# double_char("Hello World") ==> "HHeelllloo  WWoorrlldd"
+# double_char("1234!_ ") ==> "11223344!!__  """"
+str_ = input("Enter your string: ")
+print("".join(char + char for char in str_))
```
Comment (belovmd) [inline src/homework2/task8.py]: char*2

File: src/homework2/task8.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+# sieve_eratosphenes(30) == [2, 3, 5, 7, 11, 13, 17, 19, 23, 29]
+# sieve_eratosphenes(120) == [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41,
+# 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113]
+
+def sieve_eratosphenes(input_number):
+    sieve = [item for item in range(input_number + 1)]
+    # we need to exclude 1 which is not prime number from theory of school math, that's why
+    sieve[1] = 0
+    for divisor in sieve:
+        if divisor > 1:
```
Comment (belovmd) [inline src/homework2/task8.py]: if divisor would be more accurate?

**Review by belovmd — COMMENTED** [review]
good job

File: src/homework2/task8.py (hunk 1 above)
Comment (NorgeMan) [reply src/homework2/task8.py]: Done

File: src/homework2/task8.py (hunk 2 above)
Comment (NorgeMan) [reply src/homework2/task8.py]: Done

**Comment by belovmd** [comment]
9

**Pull request closed without merging**